srcpkgs/qmk/template

```diff
@@ -1,27 +1,20 @@
 # Template file for 'qmk'
 pkgname=qmk
-version=0.0.35
-revision=2
-build_style=python3-module
-hostmakedepends="python3-setuptools"
-depends="python3-appdirs
- python3-argcomplete
- python3-colorama
- python3-hjson
- python3-nose2
- flake8
- python3-yapf
- dfu-programmer
- avrdude
- dfu-util
- avr-gcc
- cross-arm-none-eabi"
+version=1.0.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-wheel"
+depends="avrdude dfu-programmer dfu-util avr-gcc hidapi"
 short_desc="CLI tool for working with QMK firmware of mechanical keyboards"
 maintainer="RinsedSloth <afernandezh@protonmail.com>"
 license="MIT"
 homepage="https://github.com/qmk/qmk_cli"
 distfiles="${PYPI_SITE}/q/qmk/qmk-${version}.tar.gz"
-checksum=e704ca31cdca264bfdd99c72a4849a9e9762043e4bf328118c49af92d1c0a3b5
+checksum=da62eec73c4548cc37b0b9be3937202dc3a301dc2f2663610ecca751a610f9ca
+
+if [ -z "$CROSS_BUILD" ]; then
```

Comment:
Making dependencies different between cross- and natively build is wrong, this should be based on whether `$XBPS_TARGET_MACHINE`'s native gcc is able to compile for the hardware.

Reply:
@Chocimier You're right of course. Better now, i hope?

Reply:
Yeah almost, we have more targets so negation is better: `armv[67]*);;` `*) depends+=`. (matching ones is a guess).

Just to be sure, do you actually use 1.0.0 built with xbps-src? What do you did about "ModuleNotFoundError: No module named 'milc'" ?

Reply:
Thank you for calling my attention on that `milc`-module. I just build this qmk package on another machine and noticed the same. Turns out that on my main system I obviously had qmk 1.0.0 installed via `pip`, so that milc was still installed.

This is however indeed a problem with regards to this PR. I could however just also make a PR for `python3-milc`. Shouldn't be to hard and wouldn't harm anyway, right?

What's your opinion on how to proceed? Should we even include this package in the repo, or just make all users install it via pip in the future? In this case, we should however remove the current `qmk` package in this repo, since it has not been compatible with the qmk firmware for quite some time...

Cheers